_config.yml

```diff
@@ -18,16 +18,11 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
-title: Your awesome title
-email: your-email@example.com
-description: >- # this means to ignore newlines until "baseurl:"
-  Write an awesome description for your new site here. You can edit this
-  line in _config.yml. It will appear in your document head meta (for
-  Google search results) and in your feed.xml site description.
-baseurl: "" # the subpath of your site, e.g. /blog
-url: "" # the base hostname & protocol for your site, e.g. http://example.com
-twitter_username: jekyllrb
-github_username:  jekyll
+title: "Cloud Intelligence Blog"
+description: "A blog showcasing thoughts, projects, and updates from Cloud Intelligence team."
+baseurl: ""
+url: "https://blog.cloudintelligence.co.za"
+
 
 # Build settings
 theme: minima
```

Comment:
Does it make more sense to create a custom theme instead of monkeypatching with css?